src/main_menu.cpp

```diff
@@ -149,7 +177,7 @@ void main_menu::print_menu( const catacurses::window &w_open, int iSel, const po
 
     int menu_length = 0;
     for( size_t i = 0; i < vMenuItems.size(); ++i ) {
-        menu_length += utf8_width( vMenuItems[i], true ) + 2;
+        menu_length += utf8_width_notags( vMenuItems[i].c_str() ) + 2;
```

Comment:
And since `shortcut_text` strips shortcut tags as mentioned above, this can be changed simply to use `shortcut_text` (and remove the addition of the first shortcut length below).